packages/remix/src/performance/client.tsx

```diff
@@ -115,6 +118,7 @@ export function withSentry<P extends Record<string, unknown>, R extends React.FC
     _useEffect(() => {
       if (activeTransaction && matches && matches.length) {
         activeTransaction.setName(matches[matches.length - 1].id);
+        activeTransaction.setMetadata({ source: 'route' });
```

Comment:
```suggestion
        activeTransaction.setName(matches[matches.length - 1].id, 'route');
```

As of https://github.com/getsentry/sentry-javascript/commit/8b4ab49b3d8b418a4728f52e32bcdb00df57c167, you can now do this in one go: https://github.com/getsentry/sentry-javascript/blob/8fe435aad2048b0cc6033eecadc4ed9c69a85091/packages/tracing/src/transaction.ts#L66-L69

Reply:
done